task/06-conditions-n-loops-tasks.js

```diff
@@ -296,7 +342,27 @@ function getDigitalRoot(num) {
  *   '{[(<{[]}>)]}' = true 
  */
 function isBracketsBalanced(str) {
-    throw new Error('Not implemented');
+    const braceObj = {
+        '[':']',
+        '{':'}',
+        '(':')',
+        '<':'>'
+    };
+    const stack = [];
+    const arrStrBalance = str.split('');
+
+    if (arrStrBalance.length % 2 !== 0) {
+        return false;
+    } else {
```

Comment:
"else" after return is redundant

Reply:
I used this 'else' after 'return false' to be sure that order of brackets is correct. For example, [')', '('] of ['{', ')'] return true if I remove the 'else' statement that is not correct.